base_kanban_stage/models/base_kanban_abstract.py

```diff
@@ -0,0 +1,112 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from openerp import api, fields, models
+
+
+class BaseKanbanAbstract(models.AbstractModel):
+    '''Inherit from this class to add support for Kanban stages to your model.
+    All public properties are preceded with kanban_ in order to isolate from
+    child models, with the exception of stage_id, which is a required field in
+    the Kanban widget and must be defined as such.'''
+
+    _name = 'base.kanban.abstract'
+    _order = 'kanban_priority desc, kanban_sequence'
+    _group_by_full = {
+        'stage_id': lambda s, *a, **k: s._read_group_stage_ids(*a, **k),
+    }
+
+    kanban_sequence = fields.Integer(
+        default=10,
+        index=True,
+        help='Order of record in relation to other records in the same Kanban'
+             ' stage and with the same priority',
+    )
+    kanban_priority = fields.Selection(
+        selection=[('0', 'Normal'), ('5', 'Medium'), ('10', 'High')],
```

Comment:
Any particular reasoning behind this priority scale?
I'm curious why not sticking to the scales using by official modules: 0,1 or 0,1,2.